charts/connect/Chart.yaml

```diff
@@ -24,7 +24,7 @@ type: application
 # The chart version and the app version are not the same and will not track
 # together. The chart version is a semver representation of changes to this
 # chart.
-version: 3.0.1
+version: 3.0.2
```

Comment:
@ooesili Should we remove so we don't release upon merge?

Reply:
Yup, good call